packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -202,7 +197,7 @@ function* editTrackAsync(action: ReturnType<typeof trackActions.editTrack>) {
 
   const track = { ...trackForEdit } as Track & TrackMetadataForUpload
   track.track_id = action.trackId
-  if (track.artwork?.file) {
+  if ('file' in track.artwork && track.artwork?.file) {
```

Comment:
I'm a little confused by this. `'file' in` will check presence of the key, and the following condition checks truthiness of the value. How is `track.artwork?.file` not sufficient to check existence of the key and truthiness of it's value?

Reply:
the 'file' in disambiguates the types here because it could be a TrackMetadataForUpload which has additional props on the `artwork` field. it's a bit messy because of how those types are laid out, but we should fix it